assets/js/blocks/cart/block.js

```diff
@@ -5,16 +5,16 @@ import { __ } from '@wordpress/i18n';
 import { useStoreCart } from '@woocommerce/base-context/hooks';
 import { useEffect } from '@wordpress/element';
 import LoadingMask from '@woocommerce/base-components/loading-mask';
-import {
-	StoreNoticesContainer,
-	SnackbarNoticesContainer,
-} from '@woocommerce/base-context';
+import { SnackbarNoticesContainer } from '@woocommerce/base-context';
```

Comment:
Do we want to move the `SnackbarNoticesContainer` too?

Reply:
Yes, I'll do it in another PR next cooldown. https://github.com/woocommerce/woocommerce-blocks/issues/7638 open to track. Thanks for pointing it out Alex.

Reply:
@alexflorisca no we don't want to do it now because as Mike said, the `SnackbarNoticesContainer` isn't location dependent, so third parties can just add a notice with type `snackbar` and add it to either the  `cart` or `checkout` contexts depending where the notice occurs.